Dockerfile

```diff
@@ -1,11 +1,26 @@
-FROM rust:1.83.0
-
-RUN apt-get update && apt install -y git curl protobuf-compiler
+# This Dockerfile describes the build process for the snapshot
+# ETL tool.
 
+FROM rust:1.83.0-bullseye AS chef
```

Comment:
Do we need a hardcoded version? Could it be just `latest`?

Reply:
this aligns with what we have in rust-toolchain.toml.

Reply:
i've no strong opinions on this, but aligning the docker base image with the toolchain version is helluva lot easier this way. i'd rather not change this tbh, the benefit to me is tentative